lib/Node/Creator.js

```diff
@@ -54,7 +54,7 @@ const NodeCreator = {
       polyTerm = this.operator('^', [polyTerm, exponent]);
     }
     if (coeff) {
-      if (NodeType.isConstant(coeff) &&
+      if (Type.isConstant(coeff) &&
```

Comment:
though I think it's more confusing here? there's definitely tradeoffs no matter what we pick I guess